Wiley Online Library.js

```diff
@@ -255,6 +255,11 @@ function scrapeBibTeX(doc, url, pdfUrl) {
 				item.title = ZU.capitalizeTitle(item.title, true);
 			}
 			
+			//date in the cochraine library RIS is wrong
+			if (ZU.xpathText(doc, '//meta[@name="citation_book_title"]/@content') == "The Cochrane Library") {
```

Comment:
Hopefully the test case below catches it if the date disappears from the meta tag